README.md

```diff
@@ -561,6 +561,59 @@ Use it to type in your key and install.
 
 Go back to Step 4A-4 and then finish with Step 4A-5.
 
+---
+## Forks
+
+### Which Fork Should I Use?
+
+If you have to ask, you're not ready for forks.
+
+Begin your research in Discord #custom-forks
+
+Please do not ask about forks outside of that channel.
+
+If you're new, please start with `commaai/master-ci` and use it for two weeks. This is the latest official version with only lateral support. This will give you a good baseline to compare the other forks to.
+
+> [!CAUTION]
+> Some forks will brick your C3X. \
+> Most forks are not made for C3X because C3X is less than a year old.
+> 
+> Some forks contain banned code. \
+> Using it will get you banned by the comma.ai company.
+> 
+> Some forks have nudgeless-lane-change. \
+> Simply clicking the turn signal will move your car to the next lane. \
+> Without any checks. \
+> Yes, it will drive into the car next to you.
+> 
+> Some forks play a blood-curdling goat scream at max volume randomly.
+> 
+> The danger in using forks is not theoretical because it has full control over your car.
+> 
+> There's a real and present danger in using forks, even on the most popular forks.
+
+If you acknowledge the warning above and are still looking to try a fork that supports TSK the following are available to install at your own risk. Please keep in mind that these are community maintained and may not stay up to date:
+
+#### Forks with Long
+
```

Comment:
Should long should be documented as not as well tested for the moment and that users looking into forks might want to use a lat-only option and one that is more well supported like `master-new` on SP (if it works?).

Reply:
Eventually, longitudinal support should be the norm and the distinction shouldn't need to be made.

Reply:
It may make sense to write that the long support is from a community-made code that's not yet upstreamed, and thus its uses have not gone through the same amount of validations that lat support has

Reply:
I mean for that matter, `optskug/SiennaFP`'s lat is also not from upstream. I guess it may make sense to note that there too